platform/darwin/src/MGLMapSnapshotter.mm

```diff
@@ -726,9 +726,23 @@ - (void)configureWithOptions:(MGLMapSnapshotOptions *)options {
 
     // Create the snapshotter
     mbgl::optional<std::string> localFontFamilyName = config.localFontFamilyName ? mbgl::optional<std::string>(std::string(config.localFontFamilyName.UTF8String)) : mbgl::nullopt;
+    mbgl::GlyphsRasterizationOptions glyphsRasterizationOptions;
+    glyphsRasterizationOptions.fontFamily = localFontFamilyName;
```

Comment:
if localFontFamilyName is `nullopt`, we should fallback to `MGLNoGlyphsRasterizedLocally`

Reply:
+1